app/models/hub_edos/person_api/v1/sis_person.rb

```diff
@@ -0,0 +1,27 @@
+module HubEdos
+  module PersonApi
+    module V1
+      class SisPerson < HubEdos::Proxy
```

Comment:
When inheriting from a class higher in the module chain, I like to be specify the starting point by notating it like this: `class SisPerson < ::HubEdos::Proxy`. Not necessary but it's slightly more specific